### PR TITLE
[Drools 5321] Failing test org.kie.hacep.LocalStorageKieSessionTest.fireUntilHaltTest on timeout

### PR DESCRIPTION
**Pull request by desmax74**
Debug on LocalMessageSystem 
@mariofusco @danielezonca

### CODE REVIEW

File: drools-ha/ha-core-infra/src/test/java/org/kie/hacep/LocalStorageKieSessionTest.java [hunk 1]
```diff
@@ -127,7 +127,7 @@ public void deleteTest() throws ExecutionException, InterruptedException {
         assertEquals((Long) 0L, session.getFactCount().get());
     }
 
-    @Test(timeout = 10000)
+    @Test(timeout = 20000)
```
Comment (mareknovotny) [inline drools-ha/ha-core-infra/src/test/java/org/kie/hacep/LocalStorageKieSessionTest.java:130]: here it timed out even for 20000ms https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/upstream-downstream/job/droolsjbpm-build-bootstrap.downstream/87/testReport/org.kie.hacep/LocalStorageKieSessionTest/fireUntilHaltTest/

Comment (desmax74) [reply drools-ha/ha-core-infra/src/test/java/org/kie/hacep/LocalStorageKieSessionTest.java:130]: Yep, I've added logs in the LocalSystem to understand where is the place of the stopped flow in this specific test

Comment (mareknovotny) [reply drools-ha/ha-core-infra/src/test/java/org/kie/hacep/LocalStorageKieSessionTest.java:130]: So what do you want to do now? merge this and wait for failure on FDB?

Comment (desmax74) [reply drools-ha/ha-core-infra/src/test/java/org/kie/hacep/LocalStorageKieSessionTest.java:130]: Jenkins execute full downstream build

**Comment by desmax74** [comment]
Jenkins execute full downstream build

**Comment by desmax74** [comment]
Jenkins execute full downstream build

**Comment by desmax74** [comment]
Jenkins execute full downstream build

**Comment by mareknovotny** [comment]
Jenkins retest this please

**Comment by mareknovotny** [comment]
Jenkins retest this please

**Pull request merged**